ethicalads-theme/templates/ea/publishers.html

```diff
@@ -134,6 +134,64 @@ <h2 class="text-center font-weight-bold mb-8">Join the only developer-focused ad
 </section>
 
 
+<!-- Subpage/topics callout
+================================================== -->
+<section class="py-10" id="audiences">
+  <div class="container">
+
+    <h2 class="text-center font-weight-bold mb-8">Developer Audiences</h2>
+    <p class="text-center lead">We focus on different developer personas using our ML-based ad targeting for each audience</p>
+
+    <div class="row">
```

Comment:
```suggestion
    <div class="row  justify-content-center">
```

This will center the columns when they aren't even.
![Screenshot from 2023-07-19 14-53-45](https://github.com/readthedocs/ethicalads.io/assets/185043/a2218669-4e48-41ee-91fb-0740522cf35e)